package.json

```diff
@@ -28,7 +28,8 @@
     "gitbook-plugin-github": "2.0.0",
     "gitbook-plugin-prism": "^2.3.0",
     "jest": "20.0.4",
-    "lerna": "2.0.0-rc.5"
+    "lerna": "2.0.0-rc.5",
+    "query-string": "^4.2.2"
```

Comment:
Since this is a monorepo, this package should be installed inside `packages/gluestick`. This `package.json` is for dependencies used to develop gluestick packages, each package have it's own `package.json` where the actual dependencies should be specified.

Reply:
Got it. Good catch! Moved this dependency to the gluestick package instead